integrated_channels/canvas/models.py

```diff
@@ -106,3 +121,97 @@ def get_content_metadata_transmitter(self):
         Return a ``CanvasContentMetadataTransmitter`` instance.
         """
         return CanvasContentMetadataTransmitter(self)
+
+
+@python_2_unicode_compatible
+class CanvasLearnerDataTransmissionAudit(models.Model):
```

Comment:
This is a hell of a nitpick, but can we be consistent on `CanvasLearner<Thing>` vs `CanvasLearnerData<Thing>` ?

Reply:
It may also be that the integrated_channels just isn't consistent on this either (I haven't looked) in which case ignore me.

Reply:
Most, if not all of my naming here was in following other integrated channels. Thinking about it, this is the learner data, so it makes sense to me to have it be `CanvasLearnerData<thing>`, but where are you seeing examples of the former?

Reply:
`class CanvasLearnerExporter` `CanvasLearnerTransmitter`.. I think those might be it. As you said it seems to follow the rest of the integrated_channels integrations so it's fine, I just don't know why someone decided to add the word 'Data' to the already very long class names, but not on the shorter ones lol.